opcua/ua/uatypes.py

```diff
@@ -409,6 +412,21 @@ def __ne__(self, other):
     def __hash__(self):
         return hash(self.__key())
 
+    def is_null(self):
+        ret = True
+        if self.NamespaceIndex != 0:
```

Comment:
If you check specs Part 3, page ~~59~~ 60 (Table 19) there is definition of null node. You can initialise null NodeId just using `NodeId()` which was already implemented :)

Regarding the NamespaceIndex citation:

> A null NodeId always has a NamespaceIndex equal to 0.

I implemented request for NamespaceIndex through BrowseName which is of type `QualifiedName` and there is NamespaceIndex inside. So the high-level interface to request the NamespaceIndex stayed the same... Check the implementation in manage_node.py of `_parse_add_args`.